bin/git-reauthor

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env bash
+
+
+init_variables() {
+  COMMAND=${0#*-}
+
+  CONFIG=false
+  ALL=false
+  unset OLD_EMAIL
+  unset CORRECT_EMAIL
+  unset CORRECT_NAME
+  TYPE='full'
+}
+
+
+usage() {
+  cat << EOF
+usage: git ${COMMAND} [<options>]
+
+Options
+    -A, --all                     rewrite all commits and tags
```

Comment:
It must be evident that this option rewrites every single commit and tag in the repo. Can we write `all` in all caps? "rewrite ALL commits and tags"